CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 ## UNRELEASED
 * **Removed**
-   * REMOVED: validity checks for historical speeds [#5087](https://github.com/valhalla/valhalla/pull/5087) 
```

Comment:
My editor is set to trim trailing white-space. It's a handful of lines that got trimmed here, so I kept them. But I can revert those if that's preferable.